src/schema.cpp

```diff
@@ -416,6 +419,95 @@ const char* MODELS_MAPPING_SCHEMA = R"(
     "additionalProperties": false
 })";
 
+const char* MEDIAPIPE_CONFIG_SCHEMA = R"({
+	"definitions": {
```

Comment:
Is there no way to reuse already defined schema for models? Otherwise we have to remember to change schema in two places.